liquibase-integration-tests/src/test/resources/liquibase/extension/testing/command/diffChangelog.test.groovy

```diff
@@ -37,11 +37,11 @@ Optional Args:
     Default: null
   driverPropertiesFile (String) The JDBC driver properties file
     Default: null
-  excludeObjects (String) Objects to exclude from diff
+  excludeObjects (String) Regular expression of columns to exclude. Defaults to null.
```

Comment:
I don't agree with this definition - it doesn't need to be a regular expression, not does it need to be a column. Plus, there is already information about the default value.
I would suggest changing it to `Objects to exclude from diff. Supports regular expressions`

The same would apply for other files, I'll comment on them so it's easier to track them all

Reply:
@tati-qalified 

On the subject of ". Defaults to null.", this is a pattern in `DiffOutputControlCommandStep.java`.
Not my original idea.

So, if you want that gone here, you have to ask for the same [here](https://github.com/liquibase/liquibase/blob/master/liquibase-standard/src/main/java/liquibase/command/core/helpers/DiffOutputControlCommandStep.java#L29):

```java
        INCLUDE_CATALOG_ARG = builder.argument("includeCatalog", Boolean.class).defaultValue(false)
                .description("If true, the catalog will be included in generated changeSets. Defaults to false.").build();
        INCLUDE_SCHEMA_ARG = builder.argument("includeSchema", Boolean.class).defaultValue(false)
                .description("If true, the schema will be included in generated changeSets. Defaults to false.").build();
        INCLUDE_TABLESPACE_ARG = builder.argument("includeTablespace", Boolean.class).defaultValue(false)
                .description("Include the tablespace attribute in the changelog. Defaults to false.").build();
```

Do you still want me to remove the " .... Defaults to [something]." ?

Reply:
I changed the wording to "Objects to exclude from Diff. Supports regular expressions. Defaults to null."